workflow-templates/hqs-build-deploy-rust-pyo3.yml

```diff
@@ -39,7 +38,7 @@ jobs:
       py_interface_folder: "PATH_TO_PYINTERFACE_FOLDER"
       deploy: true
       # Whether to build for aarch64. Can fail if there are C/C++ dependencies
-      build_for_arm: false
+      build_for_arm: true
```

Comment:
Will this break things?

Reply:
Generally I don't see problems with ARM on github in our repos, so I figured we could set it to true as default